src/main/java/net/fabricmc/mappingio/format/tiny/TinyProperties.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.mappingio.format.tiny;
+
+/**
+ * All standard properties of the Tiny format.
+ * Internally, trees use the lowest common denominator.
+ */
+public final class TinyProperties {
```

Comment:
Nit: Might want to add a private constructor to this since I merged: https://github.com/FabricMC/mapping-io/commit/806152ee3d81f7b1f4c0e8b2d3512d0f93e10530

Not that it really matters tho.

Reply:
Will do in a few minutes! 👍

Reply:
Fixed!